apps/admin/src/containers/App/Routes.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { AuthSwitch } from '@thecointech/shared/containers/AuthRoute';
+import { NotFoundPage } from '@thecointech/shared/containers/NotFoundPage';
+import { AccountId } from '@thecointech/signers';
+import { BrokerCAD } from '../BrokerCAD';
+import { TheCoin } from '../TheCoinAccount';
+
+export const AppRoutes = {
```

Comment:
porting the new routing layout to admin

Reply:
New routing layout? Is it something we will have to do everywhere?

Reply:
I tried to do it everywhere in this PR.  It's part of why it got so humungous.  (but note: 5000 LoC dropped - even most of them were in `yarn.lock`!